### ISSUE TITLE
Support passing --watch flag to limactl list

### ISSUE THREAD

**Issue by Dentrax** [lima-vm/lima#546]
We're (@developer-guy) just thinking about how this `--watch` flag fits into limactl, like how Kubernetes did for `$ kubectl get pods --watch`. Would it be good UX?

**Comment by afbjorklund** [comment]
Or `watch limactl list`

**Comment by Dentrax** [comment]
> Or `watch limactl list`

Yes, this will continuously run `limactl list` with _t_ interval. So, we will see latest sate. What I meant here: whenever any update/change/delete happen to the _instance_, we would see the update if `--watch, -w` flag passed. So, dropped an issue here to discuss whether it's worth to implement.

**Comment by jandubois** [comment]
> discuss whether it's worth to implement

I have no objection to adding it, as long as it remains reasonably simple. Are you suggesting just `--watch`, or also `--watch-only`, will the polling interval be configurable, etc.


So far I never felt a need for this, so won't be implementing it, but happy to discuss further, and review implementation.
